src/mjolnir/adminbuilder.cc

```diff
@@ -75,83 +49,47 @@ int polygondata_comparearea(const void* vp1, const void* vp2) {
   return 1;
 }
 
-std::vector<std::string> GetWkts(std::unique_ptr<Geometry>& mline) {
-  std::vector<std::string> wkts;
-
-#if 3 == GEOS_VERSION_MAJOR && 6 <= GEOS_VERSION_MINOR
-  auto gf = GeometryFactory::create();
-#else
-  std::unique_ptr<GeometryFactory> gf(new GeometryFactory());
-#endif
-
-  LineMerger merger;
-  merger.add(mline.get());
-#if GEOS_VERSION_MAJOR == 3 && GEOS_VERSION_MINOR >= 9
-  std::vector<std::unique_ptr<LineString>> merged(merger.getMergedLineStrings());
-#else
-  std::unique_ptr<std::vector<LineString*>> merged(merger.getMergedLineStrings());
-#endif
-  WKTWriter writer;
-
-  // Procces ways into lines or simple polygon list
-#if GEOS_VERSION_MAJOR == 3 && GEOS_VERSION_MINOR >= 9
-  polygondata* polys = new polygondata[merged.size()];
-#else
-  polygondata* polys = new polygondata[merged->size()];
-#endif
+std::string GetWkt(std::vector<polygon_t>& polygons) {
+  std::ostringstream wkt;
```

Comment:
it's easy to miss and not so important: but this function only returns a single WKT string now. before that was a vector of WKT strings. however, the `wkts` vector never had more than one element, so no need for a vector at all